internal/storage/v2/clickhouse/tracestore/dbmodel/trace_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2025 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package dbmodel
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+func TestFromPtrace(t *testing.T) {
+	trace := jsonToTrace(t, "ptrace.json")
+	input := FromPtrace(trace)
```

Comment:
nit: we usually try to do a roundtrip from-to-from, to ensure that the translation to dbmodel and back is lossless (as that's the primary requirement for storage format).